content/en/v1.1/fan-box.adoc

```diff
@@ -167,22 +167,22 @@ These are components that need to be purchased.  Prices fluctuate, and so are no
 | Count | Description | URL | Number in Package 
 | 1 
 | Sanyo Denki 9BMB12F201
-| https://octopart.com/9bmb12f201-sanyo+denki-24320259
+| https://octopart.com/9bmb12f201-sanyo+denki-24320259[Link]
```

Comment:
Maybe link to the main electronics BOM?  I don't think this is the right fan, but maybe it is.  I thought it ended with 202 not 201